pkg/nuctl/command/common/helpers.go

```diff
@@ -2,41 +2,14 @@ package common
 
 import (
 	"encoding/json"
-	"fmt"
 	"io"
 	"io/ioutil"
 	"os"
 
-	"github.com/nuclio/nuclio/pkg/platform"
-
 	"github.com/ghodss/yaml"
 	"github.com/nuclio/errors"
 )
 
-func FormatFunctionIngresses(function platform.Function) string {
```

Comment:
not in use and not correct, replaced with the populated `externalInvocationURLs` field on the function status.